staff/sys/migrate-vm

```diff
@@ -107,11 +106,11 @@ def shut_down_vm(vm):
             break
 
         print('waiting for shutdown...')
-        time.sleep(1)
+        time.sleep(3)
```

Comment:
This is superficial, but how many times do you really need to see `waiting for shutdown` get printed?